lib/foundation/networking/dio.dart

```diff
@@ -51,7 +53,10 @@ Dio newDio(
     },
   ));
 
-  if (supportsHttp2) {
+  // NativeAdapter only does something on Android and iOS/MacOS
+  if (Platform.isAndroid || Platform.isIOS || Platform.isMacOS) {
+    dio.httpClientAdapter = newNativeAdapter();
```

Comment:
There *shouldn't* be any issues to just using the native methods since it's *technically* what the browsers use, so we'd just be doing requests in the same way a browser does, however it'd be nice to have some more testing on this front.

Reply:
I've tested gelbooru, gelbooru_v2, shimmie2, anime-pictures and danbooru.

They're all working on Android.